# check
def test_fail(t):
    print("dir(t)", dir(t))
    fail("because")
